app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,9 @@
   </head>
 
   <body>
+    <div class='header-naviagation'>
+      <%= link_to 'Sign Up', new_user_registration_path  %>
+    </div>
```

Comment:
- [ ] Although this is more user-friendly, for now, it might make more sense to stick to the requirements shown in the wireframe. Kindly remove this link so that all you have is the appropriate form - sign up or login form. :wink: